scopes/react/component-highlighter/label/label.tsx

```diff
@@ -1,70 +1,39 @@
-import React, { useMemo, useState } from 'react';
-import { usePopper } from 'react-popper';
-import { ComponentID } from '@teambit/component-id';
-import type { CardProps } from '@teambit/base-ui.surfaces.card';
-import type { Placement, Modifier } from '@popperjs/core';
-import '@popperjs/core';
-
-import { DefaultLabel } from './default-label';
-import { ComponentLabel } from './component-label';
-import { useAnimationFrame } from '../use-animation-frame';
-
-export interface LabelContainerProps extends React.HTMLAttributes<HTMLDivElement> {
-  targetRef: HTMLElement | null;
-  offset?: [number, number];
-  placement?: Placement;
-  flip?: boolean;
-  /** continually update label position to match moving elements */
-  watchMotion?: boolean;
+import React, { useEffect, useState } from 'react';
+import classNames from 'classnames';
+import { ComponentMetaHolder } from '@teambit/react.ui.highlighter.component-metadata.bit-component-meta';
+
+import styles from './label.module.scss';
+import { ComponentStrip, ComponentStripSize } from './component-strip';
+import { OtherComponentsPopper } from './other-components';
```

Comment:
consider renaming. `OtherComponentsPopper` is confizing

Reply:
ok